### ISSUE TITLE
v2.1.18 release seems to be broken.

### ISSUE THREAD

**Issue by regadas** [lautis/refined-github-safari#22]
```
“Refined GitHub for Safari” is damaged and can’t be opened.
```

**Comment by regadas** [comment]
Btw, thanks for providing this! 🙏

**Comment by lautis** [comment]
The extension bundle has some extra files that make the code signature invalid. I'm guessing some leftover files after the refined-github paths changed.

I'll try to do a new release with clean git state.

**Comment by lautis** [comment]
Fixed in 2.1.21. Thanks for reporting!
